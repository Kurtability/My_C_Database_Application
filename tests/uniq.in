uniq a
set a 1 1 1
get a
uniq a
get a
append a 1 1 2
get a
uniq a
get a
push a 0 0 0
get a
uniq a
get a
bye
