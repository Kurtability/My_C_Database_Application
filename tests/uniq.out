> uniq a 
no such key

> set a 1 1 1 
ok

> get a
[1 1 1]

> uniq a
ok

> get a
[1]

> append a 1 1 2  
ok

> get a 
[1 1 1 2]

> uniq a
ok

> get a
[1]

> push a 0 0 0
ok

> get a
[0 0 0 1]

> uniq a
ok

> get a
[0]

> bye
bye
